Modules/Relationships/src/Relationships.Application/DomainEvents/Incoming/IdentityToBeDeleted/IdentityToBeDeletedDomainEventHandler.cs

```diff
@@ -19,7 +19,7 @@ public async Task Handle(IdentityToBeDeletedDomainEvent @event)
     {
         var relationships = await GetRelationshipsOf(@event.IdentityAddress);
 
-        NotifyRelationshipsOfPeerToBeDeleted(@event.IdentityAddress, relationships);
+        NotifyRelationshipsOfPeerToBeDeleted(@event.IdentityAddress, relationships, @event.CreationDate);
```

Comment:
Why do You use the Creation date of the event when the grace period is needed? I think, You need to change the `Relationships.Domain/DomainEvents/Incoming/IdentityToBeDeletedDomainEvent` as well as the outgoing one in the Devices Module. Then You can pass this value.

Reply:
You're absolutely right, my oversight :/ Changes made.